src/config/config.js

```diff
@@ -587,6 +587,94 @@ export const config = {
 
         'disneyaulani.com': {
             disable_venmo: true
+        },
+
+        '6abc.com': {
+            disable_venmo: true
+        },
+        
+        'abc30.com': {
+            disable_venmo: true
+        },
+        
+        'abc7.com': {
+            disable_venmo: true
+        },
+        
+        'abc7chicago.com': {
+            disable_venmo: true
+        },
+        
+        'abcnewsvideosource.com': {
+            disable_venmo: true
+        },
+        
+        'aetndigital.com': {
+            disable_venmo: true
+        },
+        
+        'araca.com.au': {
+            disable_venmo: true
+        },
+        
+        'disney.com': {
+            disable_venmo: true
+        },
+        
+        'espncustomercare.com': {
+            disable_venmo: true
+        },
+        
+        'makerstudios.com': {
+            disable_venmo: true
+        },
+        
+        'abc.org': {
+            disable_venmo: true
+        },
+        
+        'dclnews.com': {
+            disable_venmo: true
+        },
+        
+        'disneyanimation.com': {
+            disable_venmo: true
+        },
+        
+        'espncricinfo.com': {
+            disable_venmo: true
+        },
+        
+        'fort-pierce.net': {
+            disable_venmo: true
+        },
+        
+        'nationalgeographic.com': {
+            disable_venmo: true
+        },
+        
+        'nhl.tv': {
+            disable_venmo: true
+        },
+        
+        'pixar.com': {
+            disable_venmo: true
+        },
+        
+        'xgames.com': {
+            disable_venmo: true
+        },
+        
+        'espnwsummit.com': {
+            disable_venmo: true
+        },
+
+        'plus.espn.com': {
+            disable_venmo: true
+        },
+
+        'subscriptioncard.disneyplus.com': {
```

Comment:
@mstuart We really could just do the main domain as we just search for this in the domain.  Maybe just have `disneyplus.com`?

Reply:
Also,  I would have a test where the domain is not in the disallowed list, the Venmo cookie is present, and they don't have Venmo in allowed or disallowed props to ensure that the button only shows if in the allowed prop.

Reply:
Sounds good. Fixed!